Libraries/Components/FocusableWindows/FocusableWindows.windows.js

```diff
@@ -0,0 +1,270 @@
+/**
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * All rights reserved.
+ *
+ * This source code is licensed under the MIT-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Helper component for building keyboard focusable custom controls when the original native one has no such support
+ *
+ * For example, assuming a SomeComponent already exists, a custom KeyboardSupportingComponent can
+ * be built this way (at the minimum):
+ *
+ *   class KeyboardSupportingComponent extends React.Component {
+ *     render () {
+ *
+ *         styleSet = FocusableWindows.splitStyle(this.prop.styles);
+ *
+ *         return (
+ *            <FocusableWindows
+ *               ref = {(f) => this._focusable = v}
```

Comment:
f

Reply:
fixing

---
In reply to: [157098266](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r157098266) [](ancestors = 157098266)